avm/res/operations-management/solution/main.bicep

```diff
@@ -11,8 +20,11 @@ param logAnalyticsWorkspaceName string
 @description('Optional. Location for all resources.')
 param location string = resourceGroup().location
 
-@description('Optional. The product of the deployed solution. For Microsoft published gallery solution it should be `OMSGallery` and the target solution resource product will be composed as `OMSGallery/{name}`. For third party solution, it can be anything. This is case sensitive.')
-param product string = 'OMSGallery'
```

Comment:
If I see this correctly, the logic seemed to have been 'If `publisher != 'Microsoft'` the default 'OMSGallery' should be used for the `product` property. Would it make sense to keep the default, or did that default never really make much sense?

Reply:
In fact, it didn't make sense. If `publisher != 'Microsoft'`, the value must not start with `'OMSGallery'` and `publisher == 'Microsoft'`, the value was always converted to `OMSGallery/{name}`. So the default was actually never passed to the resource provider...